src/modules/healtcheck/healtcheck.controller.ts

```diff
@@ -0,0 +1,76 @@
+import { Controller, Get, Version } from '@nestjs/common';
+import {
+  ApiBearerAuth,
+  ApiOperation,
+  ApiResponse,
+  ApiTags,
+} from '@nestjs/swagger';
+import { HealtcheckService } from './healtcheck.service';
+
+@ApiBearerAuth()
+@ApiTags('healtcheck')
+//@Controller('healtcheck')
+@Controller('')
+export class HealtcheckController {
+  constructor(private readonly healtcheckService: HealtcheckService) {}
+
+  @Get('/live')
+  @Version('1')
+  @ApiOperation({ summary: 'live' })
+  @ApiResponse({ status: 200, description: 'Check if this api is running'})
+  live(): any {
+    return {
+      status: 200,
+      message: "Online",
+      datetime: new Date().toISOString()
```

Comment:
FYI, can we add this environment to the application by default
`TZ=Europe/London` - this should help with daylight savings

Reply:
Added to docker-compose. I guess API should return time in UTC timezone.